src/App.tsx

```diff
@@ -15,47 +16,111 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_BY_ALPHABET = 'aplhabet';
+const SORT_BY_LENGTH = 'length';
```

Comment:
You can create enum type here, because you have a limited number of options:
```ts
enum SortField {
  Alphabet = 'alphabet',
  Length = 'length',
  Default = '',
}

Reply:
agree with [comment](https://github.com/mate-academy/react_list-of-goods-hooks/pull/1107/files#r1258493000)